circle.yml

```diff
@@ -1,23 +0,0 @@
-machine:
```

Comment:
Since we stopped using CircleCI I guess related scripts from `package.json` can be removed as well?